Library/Formula/mpw.rb

```diff
@@ -0,0 +1,32 @@
+require "formula"
+
+class Mpw < Formula
+  homepage "http://masterpasswordapp.com"
+  url "https://ssl.masterpasswordapp.com/mpw-2.1-cli3-3-gd5ff215.tar.gz"
```

Comment:
Perfect, thanks! So future releases should be able to use the point releases directly?